DependencyInjection/Configuration.php

```diff
@@ -564,4 +523,101 @@ private function validateAuthenticationType(array $expected, array $actual, $aut
             implode(', ', $actual)
         ));
     }
+
+    /**
+     * Create configuration for cache plugin.
+     *
+     * @return NodeDefinition Definition for the cache node in the plugins list.
+     */
+    private function createCachePluginNode()
+    {
+        $builder = new TreeBuilder();
+
+        $config = $builder->root('config');
+        $config
+            ->fixXmlConfig('method')
```

Comment:
Well, as there is no XSD file, the existing XML config file will continue to work, for people lucky enough to have a working config before (it was broken when having a single method). And it will allow everyone to have a working config when using XML, for all cases.

Btw, it cannot be considered `a BC`. The sentence should be `considered a BC break`. The important work in this sentence is `break` 😄